genkit-tools/common/src/server/server.ts

```diff
@@ -79,6 +79,33 @@ export async function startServer(manager: RuntimeManager, port: number) {
     res.end();
   });
 
+  app.get('/api/latest/runtime', async (_, res) => {
```

Comment:
What is `latest` referring to here?

Reply:
Latest is referring to the fact that this endpoint will only ever return the most recent runtime info.

Reply:
Hmm, shouldn't it be flipped then? `/api/runtimes/latest`?

Reply:
Also any reason this shouldn't be part of the tRPC router?

Reply:
I ended up renaming this to `/api/sse` and made it general purpose for any events the server needs to send. I don't actually know that this will be a requirement... but... sse is limited to 6 connections for the entire browser under http(s). So better to set us up to have a single connection for the whole dev ui in the future.

As for why not tRPC, (a) my starting point is here (picking up from Shruti) https://github.com/firebase/genkit/pull/1053 but also (b) tRPC does not support SSE until v11, which is currently in "beta".